components/test_raftstore-v2/src/cluster.rs

```diff
@@ -1392,7 +1401,11 @@ impl WrapFactory {
         }
     }
 
-    fn region_id_of_key(&self, key: &[u8]) -> u64 {
+    fn region_id_of_key(&self, mut key: &[u8]) -> u64 {
+        // Strip the data prefix if the key is the data key
+        if validate_data_key(key) {
```

Comment:
When will it not be a data key?

Reply:
I think it should always be data key. I modified it in the last commit.